notebooks/tutorials/hello-syft/01-hello-syft.ipynb

```diff
@@ -11,6 +11,45 @@
     "# Hello Syft"
```

Comment:
<p>these 4 cells can be a single call to shorten things</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>now only asset and mock_df are defined, asset is needed below with syft_function</p><p> </p>